src/patternfly/components/ClipboardCopy/clipboard-copy.scss

```diff
@@ -0,0 +1,161 @@
+@import "../../patternfly-imports";
+
+.pf-c-clipboard-copy {
+  // Clipboard Copy Toggle
+  --pf-c-clipboard-copy__group-toggle--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-clipboard-copy__group-toggle--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-clipboard-copy__group-toggle--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-clipboard-copy__group-toggle--BorderTopColor: var(--pf-global--BorderColor--light-200);
+  --pf-c-clipboard-copy__group-toggle--BorderRightColor: var(--pf-global--BorderColor--light-200);
+  --pf-c-clipboard-copy__group-toggle--BorderBottomColor: var(--pf-global--BorderColor--dark);
+  --pf-c-clipboard-copy__group-toggle--BorderLeftColor: var(--pf-global--BorderColor--light-200);
+  --pf-c-clipboard-copy__group-toggle--hover--BorderBottomColor: var(--pf-global--active-color--100);
+  --pf-c-clipboard-copy__group-toggle--active--BorderBottomWidth: var(--pf-global--BorderWidth--md);
+  --pf-c-clipboard-copy__group-toggle--active--BorderBottomColor: var(--pf-global--active-color--100);
+  --pf-c-clipboard-copy__group-toggle--expanded--BorderBottomWidth: var(--pf-global--BorderWidth--md);
+  --pf-c-clipboard-copy__group-toggle--expanded--BorderBottomColor: var(--pf-global--active-color--100);
+
+  // Clipboard Copy Copy
+  --pf-c-clipboard-copy__group-copy--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-clipboard-copy__group-copy--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-clipboard-copy__group-copy--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-clipboard-copy__group-copy--BorderTopColor: var(--pf-global--BorderColor--light-200);
+  --pf-c-clipboard-copy__group-copy--BorderRightColor: var(--pf-global--BorderColor--light-200);
+  --pf-c-clipboard-copy__group-copy--BorderBottomColor: var(--pf-global--BorderColor--dark);
+  --pf-c-clipboard-copy__group-copy--BorderLeftColor: var(--pf-global--BorderColor--light-200);
+  --pf-c-clipboard-copy__group-copy--hover--BorderBottomColor: var(--pf-global--active-color--100);
+
+  // Clipboard Copy Text
+  --pf-c-clipboard-copy__group-text--PaddingTop: calc(var(--pf-global--spacer--form-element) - var(--pf-global--BorderWidth--sm));
+  --pf-c-clipboard-copy__group-text--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-clipboard-copy__group-text--PaddingBottom: calc(var(--pf-global--spacer--form-element) - var(--pf-global--BorderWidth--sm));
+  --pf-c-clipboard-copy__group-text--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-clipboard-copy__group-text--FontSize: var(--pf-global--FontSize--md);
+  --pf-c-clipboard-copy__group-text--LineHeight: var(--pf-global--LineHeight--md);
+  --pf-c-clipboard-copy__group-text--BackgroundColor: #{$pf-color-black-150};
```

Comment:
I think we should add this as another `--pf-global--BackgroundColor` var. We're also using it in nav. What do you think?

```
src//patternfly/components/ClipboardCopy/clipboard-copy.scss:  --pf-c-clipboard-copy__group-text--BackgroundColor: #{$pf-color-black-150};
src//patternfly/components/Nav/nav.scss:  --pf-c-nav__simple-list-link--hover--BackgroundColor: #{$pf-color-black-150};
src//patternfly/components/Nav/nav.scss:  --pf-c-nav__simple-list-link--active--BackgroundColor: #{$pf-color-black-150};
src//patternfly/components/Nav/nav.scss:  --pf-c-nav__simple-list-link--focus--BackgroundColor: #{$pf-color-black-150};
src//patternfly/components/Nav/nav.scss:  --pf-c-nav__simple-list-link--m-current--BackgroundColor: #{$pf-color-black-150};
```

Reply:
yes definitely I was thinking of that. I'll include that update in this PR?